sa/sa.go

```diff
@@ -1142,14 +1204,26 @@ func addIssuedNames(tx execable, cert *x509.Certificate) error {
 // |window|
 func (ssa *SQLStorageAuthority) CountFQDNSets(ctx context.Context, window time.Duration, names []string) (int64, error) {
 	var count int64
+	queryStart := ssa.clk.Now()
+	nameHashes := hashNames(names)
```

Comment:
It looks like the goal was to calculate this once and use it in both the query and the log line, but it's fairly cheap to compute; I'd just put it inline in the log line.